searchlib/src/vespa/searchlib/transactionlog/common.cpp

```diff
@@ -1,14 +1,29 @@
 // Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 
 #include "common.h"
+#include <vespa/vespalib/util/stringfmt.h>
 #include <vespa/fastos/file.h>
 
 namespace search::transactionlog {
 
 using vespalib::nbostream;
 using vespalib::nbostream_longlivedbuf;
+using vespalib::make_string;
+using std::runtime_error;
 
-int makeDirectory(const char * dir)
+namespace {
+
+void throwRangeError(SerialNum prev, SerialNum next) __attribute__((noinline));
+
+void throwRangeError(SerialNum prev, SerialNum next) {
+    if (prev < next) return;
```

Comment:
Consider removing this test which duplicates  test performed by caller.

Reply:
Fixed